docs/install-kafka-operator.md

```diff
@@ -6,7 +6,7 @@ weight: 10
 
 
 
-The operator installs version 3.1.0 of Apache Kafka, and can run on Minikube v0.33.1+ and Kubernetes 1.20.0+.
+The operator installs version 3.1.0 of Apache Kafka, and can run on Minikube v0.33.1+, Kubernetes 1.21.0+ and RedHat OpenShift 4.10, 4.11.
```

Comment:
Should we use a version  range like Kubernetes 1.21.0-1.25*

Reply:
Good Idea, I will add that.

Reply:
Fixed in https://github.com/banzaicloud/koperator-docs/compare/c586153b9586b52c2c2b77e3606f24bf75a253d8..571535c3905d7adb92eb673284f039fb5d3d26e7